src/components/Layout/index.jsx

```diff
@@ -53,12 +53,19 @@ export default Layout;
 
 
 Layout.propTypes = {
+  /** Nested structure that needs this wrapper **/
   children: t.node.isRequired,
+  /** The title that will be displayed in the header. **/
   title: t.string,
+  /** Determines whether or not the back button should be displayed in the header. **/
```

Comment:
'If a value is passed to this prop then a back button will be displayed in the header. If the value is a string then a user will go to that URL if clicked, if the value is a function then the function will be called when the back button is clicked'.

Reply:
Updated